File: server/server/Dto/AdministratorDto.cs
using server.Models.Enums;

namespace server.Dto
{
    public class AdministratorDto
    {
        public long Id { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }
        public string Email { get; set; }
        public string FirstAndLastName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Address { get; set; }
        public TypeOfUser Type { get; set; }
    }
}

File: client/src/dto/AdministratorDto.js
export default class AdministratorDto{
    constructor(id, username, password, email, firstAndLastName, dateOfBirth, address, type, image)
    {
        this.Id = id;
        this.Username = username;
        this.Password = password;
        this.Email = email;
        this.FirstAndLastName = firstAndLastName;
        this.DateOfBirth = dateOfBirth;
        this.Address = address;
        this.Type = type;
        this.Image = image;
    }
}